{"name": "邮箱输入特殊字符", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55735d7d1783 <unknown>\n#1 0x55735d4fd917 <unknown>\n#2 0x55735d53c31b <unknown>\n#3 0x55735d53c411 <unknown>\n#4 0x55735d577674 <unknown>\n#5 0x55735d55ce2d <unknown>\n#6 0x55735d5750d9 <unknown>\n#7 0x55735d55cbd3 <unknown>\n#8 0x55735d530a7b <unknown>\n#9 0x55735d53181e <unknown>\n#10 0x55735d792e98 <unknown>\n#11 0x55735d796d67 <unknown>\n#12 0x55735d7a132c <unknown>\n#13 0x55735d797993 <unknown>\n#14 0x55735d766827 <unknown>\n#15 0x55735d7bb8a8 <unknown>\n#16 0x55735d7bba36 <unknown>\n#17 0x55735d7ca5f3 <unknown>\n#18 0x7f225b994609 start_thread", "trace": "self = <test_create_member.TestCreateMember object at 0x10f7bb6a0>\ninvalid_type = 'invalidEmail', username = 'test', acctid = 'test27@qq.com'\nphone = '15650779827', email = '@#$%@qq.com', excepted = '请填写正确的邮箱地址'\ndesc = '邮箱输入特殊字符'\n\n    @allure.story(\"无效测试\")\n    @pytest.mark.parametrize(\"invalid_type, username, acctid, phone, email, excepted, desc\",\n                             OperateYaml.read_yaml(\"dates/mock_data.yaml\").get(\"qywx\").get(\"invalidCreate\"))\n    def test_invalid_create_member(self, invalid_type, username, acctid, phone, email, excepted, desc):\n        allure.dynamic.title(desc)\n        if invalid_type == \"invalidUsername\":\n            result = self.home \\\n                .click_add_by_home() \\\n                .create_member_invalid_username(username, acctid, phone, email)\n        if invalid_type == \"invalidAcctid\":\n            result = self.home \\\n                .click_add_by_home() \\\n                .create_member_invalid_acctid(username, acctid, phone, email)\n        if invalid_type == \"invalidPhone\":\n            result = self.home \\\n                .click_add_by_home() \\\n                .create_member_invalid_phone(username, acctid, phone, email)\n        if invalid_type == \"invalidEmail\":\n>           result = self.home \\\n                .click_add_by_home() \\\n                .create_member_invalid_email(username, acctid, phone, email)\n\n用户端测试/L4/tests/member_management/test_create_member.py:143: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n用户端测试/L4/page_objects/home_page.py:18: in click_add_by_home\n    self.wait_element_until_click(self._ADD_MEMBER).click()\n用户端测试/L4/page_objects/base_page.py:73: in wait_element_until_click\n    return WebDriverWait(self.driver, 10).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"65816d85666c438bcb39dbbfb6d2b22f\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x10f9550d0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55735d7d1783 <unknown>\nE       #1 0x55735d4fd917 <unknown>\nE       #2 0x55735d53c31b <unknown>\nE       #3 0x55735d53c411 <unknown>\nE       #4 0x55735d577674 <unknown>\nE       #5 0x55735d55ce2d <unknown>\nE       #6 0x55735d5750d9 <unknown>\nE       #7 0x55735d55cbd3 <unknown>\nE       #8 0x55735d530a7b <unknown>\nE       #9 0x55735d53181e <unknown>\nE       #10 0x55735d792e98 <unknown>\nE       #11 0x55735d796d67 <unknown>\nE       #12 0x55735d7a132c <unknown>\nE       #13 0x55735d797993 <unknown>\nE       #14 0x55735d766827 <unknown>\nE       #15 0x55735d7bb8a8 <unknown>\nE       #16 0x55735d7bba36 <unknown>\nE       #17 0x55735d7ca5f3 <unknown>\nE       #18 0x7f225b994609 start_thread\n\n/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:90: TimeoutException"}, "steps": [{"name": "在首页,点击添加成员", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55735d7d1783 <unknown>\n#1 0x55735d4fd917 <unknown>\n#2 0x55735d53c31b <unknown>\n#3 0x55735d53c411 <unknown>\n#4 0x55735d577674 <unknown>\n#5 0x55735d55ce2d <unknown>\n#6 0x55735d5750d9 <unknown>\n#7 0x55735d55cbd3 <unknown>\n#8 0x55735d530a7b <unknown>\n#9 0x55735d53181e <unknown>\n#10 0x55735d792e98 <unknown>\n#11 0x55735d796d67 <unknown>\n#12 0x55735d7a132c <unknown>\n#13 0x55735d797993 <unknown>\n#14 0x55735d766827 <unknown>\n#15 0x55735d7bb8a8 <unknown>\n#16 0x55735d7bba36 <unknown>\n#17 0x55735d7ca5f3 <unknown>\n#18 0x7f225b994609 start_thread\n\n", "trace": "  File \"/Users/zhangyarong/zhangyarong/student_management_system/用户端测试/L4/page_objects/home_page.py\", line 18, in click_add_by_home\n    self.wait_element_until_click(self._ADD_MEMBER).click()\n  File \"/Users/zhangyarong/zhangyarong/student_management_system/用户端测试/L4/page_objects/base_page.py\", line 73, in wait_element_until_click\n    return WebDriverWait(self.driver, 10).until(\n  File \"/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/selenium/webdriver/support/wait.py\", line 90, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1692170402150, "stop": 1692170415906}], "parameters": [{"name": "invalid_type", "value": "'invalidEmail'"}, {"name": "username", "value": "'test'"}, {"name": "acctid", "value": "'test27@qq.com'"}, {"name": "phone", "value": "'15650779827'"}, {"name": "email", "value": "'@#$%@qq.com'"}, {"name": "excepted", "value": "'请填写正确的邮箱地址'"}, {"name": "desc", "value": "'邮箱输入特殊字符'"}], "start": 1692170402150, "stop": 1692170415910, "uuid": "9dbc7153-9d19-49e3-8ed9-e3419c4ae65a", "historyId": "66f6e7b07b828b2b0c4f9d6bbbf8621f", "testCaseId": "0eaf5b13c9f9b321b15899d366539e93", "fullName": "ç¨æ·ç«¯æµè¯.L4.tests.member_management.test_create_member.TestCreateMember#test_invalid_create_member", "labels": [{"name": "epic", "value": "通讯录管理"}, {"name": "story", "value": "无效测试"}, {"name": "feature", "value": "创建通讯录成员"}, {"name": "parentSuite", "value": "ç¨æ·ç«¯æµè¯.L4.tests.member_management"}, {"name": "suite", "value": "test_create_member"}, {"name": "subSuite", "value": "TestCreateMember"}, {"name": "host", "value": "Z-melody.local"}, {"name": "thread", "value": "19149-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "ç¨æ·ç«¯æµè¯.L4.tests.member_management.test_create_member"}]}